contractions/data/slang_dict.json

```diff
@@ -1,8 +1,17 @@
 {
+    "'aight": "alright",
+    "dunno": "do not know",
+    "howdy": "how do you do",
     "ima": "I am going to",
+    "innit": "is it not",
+    "iunno": "I do not know",
+    "g'day": "good day",
     "gonna": "going to",
     "gotta": "got to",
+    "methinks": "me thinks",
+    "so're": "so are",
     "wanna": "want to",
+    "wonnot": "will not",
```

Comment:
these 3 do not seem very common? I never heard them

Reply:
So am I, since I am not a native speaker, Wikipedia is my source of truth here.

![image](https://user-images.githubusercontent.com/91830424/140738023-3cee1eae-da19-4708-ad8a-51cff31e747f.png)
![image](https://user-images.githubusercontent.com/91830424/140738076-30e0e178-05e1-4a6b-9a20-d15d9f8f39d0.png)

![image](https://user-images.githubusercontent.com/91830424/140738122-c3d1ff33-416e-4184-9a1f-e4a88eb20172.png)

Reply:
NP, https://github.com/kootenpv/contractions/pull/41

@kootenpv